promise/Promise.js

```diff
@@ -134,7 +134,6 @@ Object.assign(Promise.prototype, {
         const startTime = Date.now();
 
         return this.catch(e => {
-            trackCfg.exception = e;
             throw e;
```

Comment:
No need to catch and immediately re-throw here, although this does now raise the interesting point of what the behavior should be when you have a track statement on a promise that throws.

If you have a catchDefault (or other customized error handling that delegates to handleException) you can get a client error report.  But it does seem problematic to track as if nothing went wrong (which is what this will do).